src/seedu/addressbook/storage/jaxb/AdaptedFees.java

```diff
@@ -0,0 +1,60 @@
+package seedu.addressbook.storage.jaxb;
+
+import javax.xml.bind.annotation.XmlAttribute;
+import javax.xml.bind.annotation.XmlElement;
+import javax.xml.bind.annotation.XmlRootElement;
+
+import seedu.addressbook.common.Utils;
+import seedu.addressbook.data.exception.IllegalValueException;
+import seedu.addressbook.data.person.Fees;
+
+/**
+ * JAXB-friendly adapted tag data holder class.
+ */
+@XmlRootElement(name = "fees")
+public class AdaptedFees {
+    @XmlElement(required = true)
+    private String value;
+
+    @XmlElement(required = true)
+    private String duedate;
+
+    @XmlAttribute(required = true)
+    private boolean isPrivate;
+    /**
```

Comment:
Hmm maybe it'll be a good idea to remove the isPrivate since it should always be private